src/table/hooks/use-column-widths.ts

```diff
@@ -93,7 +93,7 @@ const useColumnWidths = (
 
   const [columnWidths, setColumnWidths] = useState(() => getColumnWidths(columns, tableWidth, getHugWidth));
 
-  useDidUpdateEffect(() => {
+  useOnPropsChange(() => {
     setColumnWidths(getColumnWidths(columns, tableWidth - yScrollbarWidth, getHugWidth));
   }, [columns, tableWidth, yScrollbarWidth]);
```

Comment:
This is a bug here. Where it needs more deps that triggers a new column widht. But will fix in another PR.